functional-tests/tests/el_balance_transfer.py

```diff
@@ -23,43 +23,38 @@ def main(self, ctx: flexitest.RunContext):
 
         source = web3.address
         dest = web3.to_checksum_address(NATIVE_TOKEN_TRANSFER_PARAMS["DEST_ADDRESS"])
-        basefee_address = web3.to_checksum_address(NATIVE_TOKEN_TRANSFER_PARAMS["BASEFEE_ADDRESS"])
```

Comment:
Very minor comment (feel free to disregard and please dont feel blocked):

could we disable or fork the test (or at least make a comment to roll back this test to the current HEAD) instead of deleting the logic with basefee asserts?